cmd/identity/batch.go

```diff
@@ -0,0 +1,280 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"archive/tar"
+	"bytes"
+	"context"
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sync/atomic"
+	"text/tabwriter"
+
+	"github.com/mum4k/termdash"
```

Comment:
Again, I disagree.

Firstly, it's not. "a lot of code"; presently, being generous, it's 30 lines of termdash-specific code.

I think it's totally valid to want feedback on the progress of the command for the same reasons we added feedback to the other identity commands. Instead of doing something similar to what was done in [dashboard.go](https://github.com/storj/storj/blob/master/cmd/storagenode/dashboard.go), which is at least the same amount of code for just the setup. I think it makes sense to use something that does all the platform specific stuff for us; Additionally, termdash is testing and maintaining that code so we don't have to.

Based on conversations I've had with multiple people, we're probably going to want to do wizards for configuration/setup and/or make our dashboard(s) interactive in the future. This seemed like the perfect opportunity to get familiar with a library like termdash which can facilitate both those things with the additional benefits that I already mentioned. 

I'm not married to termdash in particular, there are other options, some even using the same Cross-Platform core ([termbox-go](https://github.com/nsf/termbox-go/blob/master/README.md)). I just chose termdash because it seemed like a good place to start.

Reply:
termdash is 28000LOC of code, termbox is +4000LOC.